cartographer_ros/cartographer_ros/node.cc

```diff
@@ -168,25 +168,25 @@ void Node::PublishTrajectoryStates(const ::ros::WallTimerEvent& timer_event) {
     const auto& trajectory_state = entry.second;
 
     auto& extrapolator = extrapolators_.at(entry.first);
+    const auto& time = trajectory_state.data.time;
     // We only publish a point cloud if it has changed. It is not needed at high
     // frequency, and republishing it would be computationally wasteful.
-    if (trajectory_state.pose_estimate.time != extrapolator.GetLastPoseTime()) {
+    if (time != extrapolator.GetLastPoseTime()) {
+      const carto::sensor::PointCloud& original_point_cloud =
+          trajectory_state.data.range_data_in_local.returns;
```

Comment:
Isn't this making the code longer and adds another indirection? This does not seem to improve readability. Remove?